saexec/saexec_test.go

```diff
@@ -80,7 +80,7 @@ func newSUT(tb testing.TB, hooks hook.Points) (context.Context, SUT) {
 
 	wallet := saetest.NewUNSAFEWallet(tb, 1, types.LatestSigner(config))
 	alloc := saetest.MaxAllocFor(wallet.Addresses()...)
-	genesis := blockstest.NewGenesis(tb, db, config, alloc, blockstest.WithTrieDBConfig(tdbConfig))
+	genesis := blockstest.NewGenesis(tb, db, config, alloc, blockstest.WithTrieDBConfig(tdbConfig), blockstest.WithGasTarget(hooks.Target))
```

Comment:
Ahh ok, I see why `hook.Points` wouldn't work here. While it's not ideal to force using the concrete type, generalising it at this point would be YAGNI so I'm happy with this change.